2024-08-09 23:11:17,133 Minimum price: 10, Maximum price: 350
2024-08-09 23:11:17,143 Preparing sklearn pipeline
2024-08-09 23:11:17,144 Fitting
2024-08-09 23:11:47,917 Scoring
2024-08-09 23:11:48,040 Score: 0.5507180655211765
2024-08-09 23:11:48,041 MAE: 33.530625
2024-08-09 23:11:48,041 The 'last_review' column is NOT a datetime type.
2024-08-09 23:11:48,042 Exporting model
Traceback (most recent call last):
  File "/mnt/c/Users/ptiernan/Documents/GitHub/Project-Build-an-ML-Pipeline-Starter/src/train_random_forest/run.py", line 313, in <module>
  File "/mnt/c/Users/ptiernan/Documents/GitHub/Project-Build-an-ML-Pipeline-Starter/src/train_random_forest/run.py", line 115, in go
    ######################################
  File "/home/ptiernan/miniconda3/envs/mlflow-211c8fb974d6215abb361efdc7b6c4bd9a3e8dfd/lib/python3.10/site-packages/mlflow/models/signature.py", line 212, in infer_signature
    inputs = _infer_schema(model_input) if model_input is not None else None
  File "/home/ptiernan/miniconda3/envs/mlflow-211c8fb974d6215abb361efdc7b6c4bd9a3e8dfd/lib/python3.10/site-packages/mlflow/types/utils.py", line 135, in _infer_schema
    [ColSpec(type=_infer_pandas_column(data[col]), name=col) for col in data.columns]
  File "/home/ptiernan/miniconda3/envs/mlflow-211c8fb974d6215abb361efdc7b6c4bd9a3e8dfd/lib/python3.10/site-packages/mlflow/types/utils.py", line 135, in <listcomp>
    [ColSpec(type=_infer_pandas_column(data[col]), name=col) for col in data.columns]
  File "/home/ptiernan/miniconda3/envs/mlflow-211c8fb974d6215abb361efdc7b6c4bd9a3e8dfd/lib/python3.10/site-packages/mlflow/types/utils.py", line 293, in _infer_pandas_column
    raise MlflowException(
mlflow.exceptions.MlflowException: Unable to map 'object' type to MLflow DataType. object can be mapped iff all values have identical data type which is one of (string, (bytes or byterray),  int, float).
Column 'name' has 3 missing values.
Column 'last_review' has 595 missing values.
Column 'reviews_per_month' has 595 missing values.